### PR TITLE
Introduce @lvksh/logger

### LINKED ISSUE #52: [FEATURE] @lvksh/logger
**Is your feature request related to a problem? Please describe.**
Using @lvksh/logger inside of duxcore!

**Describe the solution you'd like**
I would like for it to be implemented, thank you.

**Describe alternatives you've considered**
None, its the best.

**Additional context**
We just need it.

### PR DESCRIPTION
**Pull request by lucemans**
Closes #52

### CODE REVIEW

File: core/src/index.ts [hunk 1]
```diff
@@ -13,7 +14,7 @@ enum WorkerPurpose {
 }
 
 if (cluster.isPrimary) {
-  console.log(`Primary Process`, process.pid, "is now running!");
+  log.status(`Primary Process`, process.pid.toString(), "is now running!");
```
Comment (prokopschield) [inline core/src/index.ts:17]: how about
```
log.status(`${Primary Process ${process.pid} is now starting!`);
```

Comment (prokopschield) [inline core/src/index.ts]: how about
```suggestion
  log.status(`Primary Process ${process.pid} is now running!`);
```

File: core/src/index.ts [hunk 2]
```diff
@@ -26,15 +27,15 @@ if (cluster.isPrimary) {
 
   // Connection error with master process
   socket.on("connect_error", (err) => {
-    console.log(err instanceof Error);
-    console.log(err.message);
+    log.error(err instanceof Error ? "Error" : "Not Error");
+    log.error(err.message);
```
Comment (prokopschield) [inline core/src/index.ts]:
```suggestion
    log.error(err.message);
```

**Review by prokopschield — APPROVED** [review]
Looks good to me.

**Review by HoloPanio — APPROVED** [review]
Appears alright to me

**Pull request merged**